### PR TITLE
Remove backslash of Windows Computers.

### PR DESCRIPTION
**Pull request by ErfolgreichCharismatisch**
Remove backslash of Windows Computers.

### CODE REVIEW

File: TTS/utils/io.py [hunk 1]
```diff
@@ -43,7 +33,15 @@ def load_config(config_path: str) -> AttrDict:
...
             data = yaml.safe_load(f)
     else:
-        data = read_json_with_comments(config_path)
+        # fallback to json
+        with open(config_path, "r") as f:
+            input_str = f.read()
+        # handle comments
+        input_str = re.sub(r'\\\n', '', input_str)
+        input_str = re.sub(r'//.*\n', '\n', input_str)
+        input_str = input_str.replace('\\','/')
```
Comment (ErfolgreichCharismatisch) [inline TTS/utils/io.py:42]: Right here.

Comment (erogol) [reply TTS/utils/io.py:42]: right here what exactly?

Comment (erogol) [reply TTS/utils/io.py:42]: and why this PR? I don't see it is fixing anything? If there is pls fix the thing without removing the original code.

Comment (ErfolgreichCharismatisch) [reply TTS/utils/io.py:42]: input_str = input_str.replace('\\','/')
is resolving the backslash issue.

Comment (erogol) [reply TTS/utils/io.py:42]: now I see. Can you fit it without removing the function?

Comment (ErfolgreichCharismatisch) [reply TTS/utils/io.py:42]: What do you mean?

**Comment by erogol** [comment]
I close this due to inactivity.

**Pull request closed without merging**